lib/start_game.rb

```diff
@@ -0,0 +1,96 @@
+class StartGame
+  def initialize(cpu_board, player_board)
+    @cpu_board = cpu_board
+    @player_board = player_board
+  end
+
+  def start
+  #  intro
+  #  user_ready?
+  #  comp_ship_placement
+  #  cpu_board.board_render(true)
+    player_ship_placement
+    player_board.board_render(true)
+  end
```

Comment:
We are adding and commenting out different methods in order to test the methods within the runner file.  These test require user input so we run the method numerous times to ensure it's working properly.